packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -442,6 +442,15 @@ const TRANSFORMS = {
       startTime: metrics && toTimestamp(metrics.start_time),
       secureBoot: obj.platform.secureboot === 'true',
       suspendSr: link(obj, 'suspend_SR'),
+      supportsBios: (() => {
+        const vmRecommendations = obj.recommendations
+        const parsedRecommendations = parseXml(vmRecommendations)
+        const supportsBios = parsedRecommendations?.restrictions?.restriction.find(
+          restriction => restriction.field === 'supports-bios'
+        )?.value
+
+        return supportsBios === 'yes' ? true : supportsBios === 'no' ? false : undefined
+      })(),
```

Comment:
Use the field `VM.supports_bios` available since XCP-ng 8.3

Reply:
I don't see this field in th XAPI doc.

Reply:
@Pizzosaure I think I was mistaken and that your approach is the correct one.